<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shoppinggui</class>
 <widget class="QMainWindow" name="shoppinggui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>shoppinggui</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="shoppingListWidget"/>
    </item>
    <item>
     <widget class="QPushButton" name="deteleButton">
      <property name="text">
       <string>DELETE</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="filterLabel">
        <property name="text">
         <string>Filter by category</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="filterLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="filterButton">
        <property name="text">
         <string>FILTER</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>523</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
